grove/bernstein_vazirani/bernstein_vazirani.py

```diff
@@ -179,17 +178,30 @@ def run(self, cxn, bitstring_map):
         self.bv_circuit = self._create_bv_circuit(bitstring_map)
 
         # find vector by running the full bv circuit
-        results = cxn.run_and_measure(self.bv_circuit, self.computational_qubits)
-        bv_vector = results[0][::-1]
+        full_circuit = Program()
+        full_ro = full_circuit.declare('ro', 'BIT', len(self.computational_qubits) + 1)
+        full_circuit += self.bv_circuit
+        full_circuit += [MEASURE(qubit, ro) for qubit, ro in zip(self.computational_qubits,
+                                                                 full_ro)]
+        full_circuit.wrap_in_numshots_loop(1)
+        full_executable = qc.compiler.native_quil_to_executable(full_circuit)
+        full_results = qc.run(full_executable)
+        bv_vector = full_results[0][::-1]
```

Comment:
worth measuring into classical addresses indexed in a descending order so you don't have to flip the bitstring?

Reply:
maybe not worth it